app/src/debug/google-services.json

```diff
@@ -36,7 +36,38 @@
           "status": 2
         }
       }
+    },
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:235314010044:android:95198efc499017133fbdb9",
+        "android_client_info": {
+          "package_name": "org.simple.clinic.tamilnadu.debug"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "235314010044-5cs4deattnhsdcs4ooljphm0fla9vjvh.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyDtpDbqQ-TXRED-gdOhbRbgQIOicyGAiaM"
+        }
+      ],
+      "services": {
+        "analytics_service": {
+          "status": 1
+        },
+        "appinvite_service": {
+          "status": 1,
+          "other_platform_oauth_client": []
+        },
+        "ads_service": {
+          "status": 2
+        }
+      }
     }
   ],
   "configuration_version": "1"
-}
\ No newline at end of file
+}
```

Comment:
Now that we are not doing a debug variant for Tamil Nadu app, we can discard this change.

Reply:
Yes I thought about this and I saw that we have google.services.json for the other variants that we're filtering as well. Why is that?

Reply:
Those are for release variants. Basically we don't have multiple debug variants. We only have multiple release variants that we publish to Google Play.